eval_agent.py

```diff
@@ -36,121 +31,74 @@ def add_path(path):
     from Evaluator import *
     from utils import *
 except ModuleNotFoundError:
-    print('Object Detection Library not initialized correctly')
+    print('Object Detection Library not initialized correctly.')
```

Comment:
I would strongly prefer to just let it fail. Why catching an error and printing a warning if the script will then fail after an hour. Either remove the try catch or append the actual error to the message and then kill the program.

Reply:
In general there is no clear path to the detection metrics at the moment. There should be hint in the README, also the detection_metrics have the dependency cv2 (opencv-python) which needs to be either added to the requirements or resolved some other way.

Reply:
And couldn't the metrics just be included from the files with the path instead of loading at as a module beforehand?

Reply:
1) Importing via fully qualified module paths (i.e. detection_metrics.module) is not possible because the library imports like this internally `from BoundingBox import *` - otherwise all files would need to be moved into the root folder.
2) The try-catch gets around the opencv-python dependency, because we don't actually use any of the library functions that depend on it. It could be removed, but then we need to add opencv-python to the requirements.txt.

Honestly, it's not perfect, but I would keep it like this for now. Maybe add an explicit check so the script fails if detection_metrics is not present while keeping the ModuleNotFound try-catch for python-opencv.